javatests/google/registry/flows/domain/DomainCreateFlowTest.java

```diff
@@ -437,6 +437,18 @@ public void testSuccess_generalAvailability_ignoresEncodedSignedMark() throws Ex
   }
 
   @Test
+  public void testSuccess_superuser_ignoresEncodedSignedMark() throws Exception {
```

Comment:
Ok, fine, though I think there's an argument for only naming the test based on specifically what the test actually checks (that mismatches are ignored for superuser) even if that's supposed to represent something broader (that all SMD verification failures are ignored for superuser).

However, the point was more that it should communicate somehow that it's just the verification/validation being skipped.  IMO a test that checks that the SMD is entirely ignored for superuser would be more like checking that the created domain doesn't have an smdId field, even if the SMD was valid.  Right now, to me the test naming suggests the latter check (that valid SMDs are ignored) rather than checking that invalid SMDs are not rejected.